dd_launch

```diff
@@ -36,14 +36,13 @@ print "Liftoff!".
 wait until verticalSpeed > 10.
 
 local lock lookAt to heading(launchHeading, 90).
-local lock lookUp to heading(launchHeading, -45).
-lock steering to lookdirup(lookAt:vector, lookUp:vector).
+lock steering to lookdirup(lookAt:vector, SHIP:UP:vector).
 
 if body:atm:exists {
   wait until verticalSpeed > 50.
 
   print "Initiating gravity turn.".
-  lock targetPitch to 90 - 90 * min(1, apoapsis / (body:atm:height - 10000)).
+  lock targetPitch to 90 - 90 * min(1, altitude / (body:atm:height - 10000)).
```

Comment:
no, i really mean to target apo here.  i've tried altitude (and averaging altitude and apo).

the intent is to level out while still inside very thin atmosphere. this lets us take advantage of the oberth effect a little bit, and keep our circularization burn low. it also enables coasting to apo without having to reboost.

i know it's not a typical ksp launch or the most efficient, but i've enjoyed using it recently.